sdk/firstload/package.json

```diff
@@ -41,8 +41,8 @@
 	"exports": {
 		"types": "./dist/firstload/src/index.d.ts",
 		"import": "./dist/index.es.js",
-		"require": "./dist/index.cjs.js",
-		"node": "./dist/index.cjs.js",
+		"require": "./dist/index.cjs",
```

Comment:
Remix did NOT like importing `index.cjs.js`. It demanded `index.cjs` instead.

This change does not appear to break our Next.js implementation. I'm inclined to believe that switching to `index.cjs` is the right move.

Reply:
It worked!